web/src/features/PreflightChecks/api/getPreflightResult.tsx

```diff
@@ -135,6 +135,9 @@ function flattenPreflightResponse({
     shouldShowConfirmContinueWithFailedPreflights:
       !response?.preflightResult?.skipped && // not skipped
       (hasFailureOrWarning(response) || hasPreflightErrors(response)), // or it has errors
+    shouldShowRerunPreflight:
+      Boolean(response?.preflightResult?.result) || // not running
+      response?.preflightResult?.skipped, // not skipped
```

Comment:
so the original logic was 
`!this.props.fromLicenseFlow && stopPolling`. (see screenshot in previous comment - it's a big ternary) 

- `fromLicenseFlow` is still used for now in the component above on 222
- `stopPolling` looked like this: 
https://github.com/replicatedhq/kots/blob/af7db8652d2e24b4022b0eef939c6963fbd7712d/web/src/components/PreflightResultPage.tsx#L377

- `preflightResultData?.result` was basically checking if the the preflight result had any data. we do the same thing here. 

- we're just pulling `skipped` right off the response and ORing it like the old `stopPolling` worked
https://github.com/replicatedhq/kots/blob/af7db8652d2e24b4022b0eef939c6963fbd7712d/web/src/components/PreflightResultPage.tsx#L376